x/concentrated-liquidity/incentives.go

```diff
@@ -253,6 +254,32 @@ func (k Keeper) GetIncentiveRecord(ctx sdk.Context, poolId uint64, denom string,
 	return incentiveStruct, nil
 }
 
+// nolint: unused
+// setUpTimeTracker sets the uptime tracker in state
+func (k Keeper) setUpTimeTracker(ctx sdk.Context, poolId uint64, tickIndex int64, uptimeTracker model.UptimeTracker) {
+	store := ctx.KVStore(k.storeKey)
+	key := types.KeyUptimeTracker(poolId, tickIndex)
+	osmoutils.MustSet(store, key, &uptimeTracker)
+}
+
+// GetUpTimeTracker gets the uptime tracker corresponding to the passed in values from store
+func (k Keeper) GetUpTimeTracker(ctx sdk.Context, poolId uint64, tickIndex int64) (model.UptimeTracker, error) {
```

Comment:
nit: uncapitalize the T for naming consistency (for both `GetUpTimeTracker` and `setUpTimeTracker`)